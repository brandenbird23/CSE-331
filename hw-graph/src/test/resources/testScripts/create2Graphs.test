# Create 2 Graphs Test File
# Test to create 2 graphs
CreateGraph graph1
CreateGraph graph2

# Print the nodes in the graph
ListNodes graph1
ListNodes graph2