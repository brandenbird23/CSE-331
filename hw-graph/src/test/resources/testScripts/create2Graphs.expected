# Create 2 Graphs Expected File
# Test to create 2 graphs
created graph graph1
created graph graph2

# Print the nodes in the graph
graph1 contains:
graph2 contains: